### PR TITLE
Update CascLib

### PR DESCRIPTION
**Pull request by ChipLeo**

### CODE REVIEW

**Comment by Gooyeth** [comment]
Upgrade: https://github.com/TrinityCore/TrinityCore/blob/6.x/dep/PackageList.txt

**Comment by jackpoz** [comment]
The upgraded CascLib hash version should be in the commit log too, github link is fine

**Comment by ChipLeo** [comment]
Updated with https://github.com/ladislav-zezula/CascLib/commit/5d3789af3435534c288c2145e158d422651c7fe1

**Comment by jackpoz** [comment]
is this a blind update or it has been tested properly ?

**Comment by ChipLeo** [comment]
Tested, extract enGB, enUS DBC and maps.

File: src/tools/connection_patcher/Program.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ namespace Connection_Patcher
             if (!fs::exists(modulePath))
                 fs::create_directories(modulePath);
 
-            if (fs::exists(modulePath / modulePath))
+            if (fs::exists(modulePath / moduleName))
```
Comment (DDuarte) [inline src/tools/connection_patcher/Program.cpp]: Change added in 8bbd9133d57fc34b77544cbdd59526ed9ccaa607

File: src/tools/connection_patcher/Program.cpp [hunk 2]
```diff
@@ -251,7 +251,7 @@ int main(int argc, char** argv)
 
                 do_module<Patches::Windows::x86, Patterns::Windows::x86>
                     ( "8f52906a2c85b416a595702251570f96d3522f39237603115f2f1ab24962043c.auth"
-                    , std::wstring(commonAppData) + std::wstring(L"/Blizzard Entertainment/Battle.net/Cache/")
+                    , std::wstring(commonAppData) + std::wstring(L"\\Blizzard Entertainment\\Battle.net\\Cache\\")
```
Comment (DDuarte) [inline src/tools/connection_patcher/Program.cpp]: Is this actually required?

**Comment by DDuarte** [comment]
Still missing updates to PackageList.txt

File: src/tools/connection_patcher/Program.cpp (hunk 2 above)
Comment (ChipLeo) [inline src/tools/connection_patcher/Program.cpp]: Its work, but showing like "C:\ProgramData/Blizzard...

**Pull request closed without merging**